recipes/xoa/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "xoa" %}
+{% set version = "0.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xoa-{{ version }}.tar.gz
+  sha256: ff07013caa0843f16b8b13e0e63b6e8210a7e5c38c0edfab94ded262a2971b02
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools >=40.4
+    - setuptools-scm
+  run:
+    - appdirs
+    - cmocean
+    - configobj
+    - netcdf4
+    - numba
+    - pytest
```

Comment:
Probably shouldn't have `pytest` listed as a dependency, but that would need to be fixed upstream since it's incorrectly listed there.